src/en/reference-environment-variables.md

```diff
@@ -15,33 +15,22 @@ Use header tags so we can link to these variables individually.
 These variables are available on the juju client in order to change its default
 behavior.
 
-#### JUJU_ENV
 
-The current environment that juju will use by default. Juju has three ways it
-will  determine current environment, in descending order:
+#### JUJU_DATA
 
-1. Check to see if `JUJU_ENV` is set.
-```
-echo $JUJU_ENV
-local
-```
+This sets the path where Juju will look for its configuration files. You do not
+need to set this - by default Juju follows XDG guidelines and on Linux systems
+it will use the path:
 
-2. Read the `$JUJU_HOME/current-environment` file, which is set by the
-`juju switch` command.
-```
-juju switch amazon
-cat ~/.juju/current-environment
-amazon
+```no-highlight
+~/.local/share/juju
 ```
 
-3. Read `$JUJU_HOME/environments.yaml` and use the default environment.
-```yaml
-default: local
-```
+#### JUJU_HOME (Deprecated)
```

Comment:
As there is no upgrade path for 2.0 and won't be for some time, the reality is that many users will be running both 1.x and 2.x for some time. There are other docs detailing how to use both side by side, but since it is likely that this may be set to support a 1.x install, it doesn't hurt to have it documented here imo